charts/operator-wandb/values.yaml

```diff
@@ -1288,6 +1288,175 @@ frontend:
           cpu: 100m
           memory: 512Mi
 
+metric-observer:
+  install: false
+  imagePullSecrets: *imagePullSecrets
+  autoscaling:
+    horizontal:
+      enabled: true
+      maxReplicas: 3
+      minReplicas: 1
+  image:
+    repository: wandb/megabinary
+    tag: latest
+  env:
+    A_KAFKA_CLIENT_PASSWORD:
+      valueFrom:
+        secretKeyRef:
+          name: '{{ include "wandb.kafka.passwordSecret" . }}'
+          key: '{{ include "wandb.kafka.passwordSecret.passwordKey" . }}'
+          optional: true
+    A_REDIS_PASSWORD:
+      valueFrom:
+        secretKeyRef:
+          name: '{{ include "wandb.redis.passwordSecret" . }}'
+          key: "{{ .Values.global.redis.secret.secretKey }}"
+          optional: true
+    A_MYSQL_PASSWORD:
+      valueFrom:
+        secretKeyRef:
+          name: '{{ include "wandb.mysql.passwordSecret" . }}'
+          key: "{{ .Values.global.mysql.passwordSecret.passwordKey }}"
+    GORILLA_RUN_STORE:
+      value: '{{ include "wandb.mysql" . | trim }}'
+    GORILLA_METADATA_STORE:
+      value: '{{ include "wandb.mysql" . | trim }}'
+    GORILLA_PARQUET_LIVE_HISTORY_STORE:
+      value: '{{ include "wandb.mysql" . | trim }}'
+    GORILLA_ANALYTICS_SINK:
+      value: '{{ include "wandb.mysql" . | trim }}'
+    GORILLA_HISTORY_STORE:
+      value: 'http://{{ .Release.Name }}-parquet:8087/_goRPC_,{{ include "wandb.mysql" . | trim }}'
+    GORILLA_METADATA_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    REDIS:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_AUDITOR_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_SETTINGS_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_CACHE:
+      value: '{{ include "wandb.redis.connectionString" . | trim }}'
+    GORILLA_FILE_HOST:
+      value: "{{ .Values.global.host }}"
+    OPERATOR_ENABLED:
+      value: "true"
+    LOGGING_ENABLED:
+      value: "true"
+    GORILLA_ONPREM:
+      value: "true"
+    GORILLA_FILE_STORE:
+      value: '{{ (include "wandb.bucket" . | fromYaml).url }}'
+    GORILLA_STORAGE_BUCKET:
+      value: '{{ (include "wandb.bucket" . | fromYaml).url }}'
+    GORILLA_TRACER:
+      value: '{{ include "wandb.otelTracesEndpoint" . | trim }}'
+
+    ### I'm unsure the following are necessary
```

Comment:
these seem to be involved in otel/tracing things? I'd definitely like to have that enabled, but wasn't entirely sure what was going on with these

Reply:
this, and possibly some other values defined here is actually included in the gorilla configMap included in your envFrom later.  only you can know what values are actually used within the application though

Reply:
okay, so it sounds like it'd make sense to drop these then, so I'll see if I'm able to do that. Is there a way to see where those are defined?

Reply:
okay, I've checked the values I could see getting set in the gorilla.yaml template for the configMap, and removed the overlaps.